packages/react-native-auth/src/hooks/useQRCodeString/__tests__/__snapshots__/useQRCodeString.spec.ts.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`useQRCodeString behaves as expected in the happy path 1`] = `
+"                             
+                             
+    █▀▀▀▀▀█ ▀██▀▀ █▀▀▀▀▀█    
+    █ ███ █ ▄▀▀█▀ █ ███ █    
+    █ ▀▀▀ █ █  ▄▀ █ ▀▀▀ █    
+    ▀▀▀▀▀▀▀ █ ▀▄█ ▀▀▀▀▀▀▀    
+    ▀▄  █ ▀▀█▄██▄▀▀█▀█▄▄▀    
+    ▀ ▀ █▀▀ ██▀▀ ▄█▀█  ▀     
+      ▀ ▀ ▀▀▄▄█ ███  ▀▄ ▀    
+    █▀▀▀▀▀█ ▀▄▀▄█▀ ▄  ▀█     
+    █ ███ █ ▀▄ █▄ ▀▀▀█▄▄█    
+    █ ▀▀▀ █  ▄█▀ ▄█▀█▀       
+    ▀▀▀▀▀▀▀ ▀ ▀ ▀▀▀ ▀▀  ▀    
+                             
+                             "
+`;
+
+exports[`useQRCodeString does not call \`onSuccess\` if it is not a function 1`] = `
+"                             
+                             
+    █▀▀▀▀▀█ ▀██▀▀ █▀▀▀▀▀█    
```

Comment:
What does this snapshot mean?

Reply:
It means that we have a stable value to diff against in the unit test 😅